phasespace/fulldecay/mass_functions.py

```diff
@@ -0,0 +1,64 @@
+import tensorflow as tf
+import zfit
+import zfit_physics as zphys
+
+
+# TODO refactor these mass functions using e.g. a decorator.
+#  Right now there is a lot of code repetition.
+def gauss(mass, width):
+    particle_mass = tf.cast(mass, tf.float64)
+    particle_width = tf.cast(width, tf.float64)
+
+    def mass_func(min_mass, max_mass, n_events):
+        min_mass = tf.cast(min_mass, tf.float64)
+        max_mass = tf.cast(max_mass, tf.float64)
+        pdf = zfit.pdf.Gauss(mu=particle_mass, sigma=particle_width, obs="")
+        iterator = tf.stack([min_mass, max_mass], axis=-1)
+        return tf.vectorized_map(
+            lambda lim: pdf.sample(1, limits=(lim[0], lim[1])), iterator
+        )
+
+    return mass_func
+
+
+def breitwigner(mass, width):
+    particle_mass = tf.cast(mass, tf.float64)
+    particle_width = tf.cast(width, tf.float64)
+
+    def mass_func(min_mass, max_mass, n_events):
+        min_mass = tf.cast(min_mass, tf.float64)
+        max_mass = tf.cast(max_mass, tf.float64)
+        pdf = zfit.pdf.Cauchy(m=particle_mass, gamma=particle_width, obs="")
+        iterator = tf.stack([min_mass, max_mass], axis=-1)
+        return tf.vectorized_map(
+            lambda lim: pdf.sample(1, limits=(lim[0], lim[1])), iterator
+        )
+
+    return mass_func
+
+
+def relativistic_breitwigner(mass, width):
+    particle_mass = tf.cast(mass, tf.float64)
+    particle_width = tf.cast(width, tf.float64)
+
+    def mass_func(min_mass, max_mass, n_events):
+        min_mass = tf.cast(min_mass, tf.float64)
+        max_mass = tf.cast(max_mass, tf.float64)
+        pdf = zphys.pdf.RelativisticBreitWigner(
+            m=particle_mass, gamma=particle_width, obs=""
+        )
+        iterator = tf.stack([min_mass, max_mass], axis=-1)
+        # TODO this works with map_fn but not with vectorized_map for some reason.
+        #  Does not work for e.g., zfit.pdf.CrystalBall either
+        return tf.map_fn(
+            lambda lim: pdf.sample(1, limits=(lim[0], lim[1])).unstack_x(), iterator
+        )
+
+    return mass_func
+
+
+_DEFAULT_CONVERTER = {
+    "gauss": gauss,
+    "BW": breitwigner,

```

Comment:
What about "bw" and "relbw"?

Reply:
Do you mean that "bw" and "relbw" should be additional aliases for Breit-Wigner and relativistic Breit-Wigner, or that they should be renamed?

Reply:
I would suggest to rename. The argument being that lowercase seems more consistent. Sure, relbw is quite short, but rel-BW does not seem to add a lot. Or maybe rel-bw?

But for sure, use the lowercase anyway and check against `input.lower()`, so that the capitalization doesn't matter (do you agree?)

Reply:
I think I simply took the `rel-BW` name from a [comment in the issue](https://github.com/scikit-hep/decaylanguage/issues/86#issuecomment-551312687), so i do not have a strong opinion about the names. "bw" and "relbw" seem reasonable. Adding `input.lower()` seems unnecessary but if you prefer to include that feature, I can of course add it.

Reply:
The `input.lower()` would mostly be if we'd use capitalized names, but other than that it's also fine. So fine with me to not add it

Reply:
Fixed in 33ef57b88ac8afdf3363447ec4f377527802fdd5